remotes/docker/resolver.go

```diff
@@ -584,7 +584,7 @@ func requestFields(req *http.Request) logrus.Fields {
 			fields[field] = v
 		}
 	}
-
+	fields["http.version"] = req.Proto
```

Comment:
what is this?

Reply:
Extra logs. Just logs the http protocol version i.e "HTTP/1.1 OR HTTP/2.0"

Reply:
where is it used?

Reply:
It's containerd log output that you can view if its run in the foreground, it can be removed if youd like.